app/helpers/sessions_helper.rb

```diff
@@ -8,7 +8,7 @@ def current_user
       @current_user ||= User.find_by id: session[:user_id]    
     elsif user_id = cookies.signed[:user_id]
       user = User.find_by id: user_id
-      if user&.authenticated? cookies[:remember_token]
+      if user&.authenticated? :remember, cookies[:remember_token]
```

Comment:
```suggestion
      if user&.authenticated?(:remember, cookies[:remember_token])
```
I'm not sure if you don't have the parenthesis